evaluation/tasks/tydiqa_primary/tydiqa_primary.py

```diff
@@ -16,30 +17,31 @@
     {{"\n"}}{{context}}
     {%- endif -%}
     {{"\n"}}Answer: 
-    """
+    """  # noqa W291
```

Comment:
Although the trailing whitespace after `Answer:` is understandable, `load_dataset()` may still remove it with tokenizer. See also the test.